task/net/src/bsp/gimletlet_mgmt.rs

```diff
@@ -28,64 +28,77 @@ use vsc85xx::VscError;
 
 task_slot!(USER_LEDS, user_leds);
 
-#[derive(Copy, Clone, Eq, PartialEq)]
+#[derive(Copy, Clone, Eq, PartialEq, counters::Count)]
 enum Trace {
+    #[count(skip)]
     None,
     BspConfigured,
 
     KszErr {
+        #[count(children)]
         err: KszError,
     },
+    #[count(skip)]
     Ksz8463Status {
         port: u8,
         status: u16,
     },
+    #[count(skip)]
     Ksz8463Control {
         port: u8,
         control: u16,
     },
+    #[count(skip)]
     Ksz8463Counter {
         port: u8,
         counter: MIBCounterValue,
     },
 
+    #[count(skip)]
     Vsc8552Status {
         port: u8,
         status: phy::standard::MODE_STATUS,
     },
+    #[count(skip)]
     Vsc8552MacPcsStatus {
         port: u8,
         status: phy::extended_3::MAC_SERDES_PCS_STATUS,
     },
+    #[count(skip)]
     Vsc8552MacPcsControl {
         port: u8,
         control: phy::extended_3::MAC_SERDES_PCS_CONTROL,
     },
+    #[count(skip)]
```

Comment:
We skip counting the various status and control register ringbuf entries because they don't _really_ seem to represent a countable event, just the state of a register, and skipping all of them lets us avoid several words of counters. Instead, we just count the number of error event variants.

Reply:
Consider making this a comment in the source code.